example/ios/Runner/AppDelegate.swift

```diff
@@ -1,12 +1,64 @@
 import UIKit
 import Flutter
+import workmanager
 
 @UIApplicationMain
 
 @objc class AppDelegate: FlutterAppDelegate {
+    
     override func application(_ application: UIApplication, didFinishLaunchingWithOptions launchOptions: [UIApplicationLaunchOptionsKey: Any]?) -> Bool {
+        
+        SwiftWorkmanagerPlugin.clearStorage()
```

Comment:
🤔 ? 
Explain

Reply:
If the plugin user runs the project multiple times, nothing prevents him from tapping the "Simulate Background Fetch" button without first initialising the Flutter background service, and he wouldn't receive an error since the Callback handle has already been stored in UserDefaults through a previous call.
Clearing SwiftWorkManagerPlugin's storage ensures the app is ran from a clean state every time.

Reply:
But, that's what we want right.  
It's ok for the implementer to only register the `callbackDispatcher` function once.

Reply:
Of course, but it would cause a failure if UserDefaults is populated with a different entryPoint name (for example because the name has changed).

Reply:
That's true! However should we catch such an error in the example?

Reply:
Which means you "pollute" (big quotes on this one) the SharedPrefs with something that cannot be removed ; if we push it further, it also prevents unit testing this feature correctly

Reply:
What do you mean?

Reply:
Typically, you'd want to ensure that the SharedPrefs (or any other storage injected in the test) actually contains the callback handle once the foreground channel has been initialised.
The first test will pass ; but if changes are brought to the code and this callback handle is no more correctly stored in the SPs, running the test again will result in a false positive, since the SPs weren't cleaned

Reply:
But that's testing? 
I don't see how this is related tbh.

You'd have some sort of cleanup of tear down of a test that's reset it to a default state.